### ISSUE TITLE
Enhancement: CSV File Loader

### ISSUE THREAD

**Issue by PCMonsterx** [theUpsider/ComfyUI-Styles_CSV_Loader#2]
Hello I am using the node within ComfyUI and think it is a great idea. The only issue I have, like the Prompt Style Chooser in WAS Suite, I would like to have this same power using other CSV files. The best option for this I think would be to add a section within the node where you are able to input the name of the CSV file you want to reference. For example I have copied the Custom node and replaced all references to "styles" with "settings" and now I have a settings.csv. This is a workable approach but would be much easier if there was only one node where the CSV can be referenced. I'm not sure if there is a limit in the functionality of ComfyUI when it comes to this but would be a much better approach in my opinion.

If there are limitations, maybe extra nodes can be added as seen below for basic things you would include in a prompt.

- Style
- Setting
- Lighting
- Composition
- Artist
- etc...

![image](https://github.com/theUpsider/ComfyUI-Styles_CSV_Loader/assets/58462961/837af7cb-3859-4dd2-bf3a-225f2f013c92)
![image](https://github.com/theUpsider/ComfyUI-Styles_CSV_Loader/assets/58462961/598ae3e8-7041-4a1b-a429-0fcb5000a698)

Thank you for the node regardless, saved me the time of trying to pull the code from the WAS Suite and figuring it out myself.

**Comment by PCMonsterx** [comment]
So I went ahead with the second suggestion I posted above. Feel free to implement it into the base model if you would like. The only other change I would like is to actually show the positive prompt and negative prompt below the name. I'm not sure how to implement that though.. so if you do I would be great full.

The folder with the CSV files has been moved to the `ComfyUI\custom_nodes\ComfyUI-CSV_Loader` folder to keep everything contained and updated everything to be CSV Loader instead of Style CSV Loader. The CSV files include artists.csv, artmovements.csv, characters.csv, composition.csv, lighting.csv, negative.csv, positive.csv, settings.csv, and styles.csv. All have some preloaded selections but can always be updated in the CSV files.
[ComfyUI-CSV_Loader.zip](https://github.com/theUpsider/ComfyUI-Styles_CSV_Loader/files/13233721/ComfyUI-CSV_Loader.zip)

https://github.com/theUpsider/ComfyUI-Styles_CSV_Loader/assets/58462961/9d6c7fbf-9ae4-4e90-a2cc-cc98aa6cc0f8
